src/main/java/com/rpg2014/wrappers/SpidermanEC2Wrapper.java

```diff
@@ -28,7 +51,7 @@
     private String oldSnapshotId;
 
     private SpidermanEC2Wrapper() {
-        this.ec2Client = Ec2Client.create();
+        this.ec2Client = Ec2Client.builder().region(Region.US_EAST_1).build();
```

Comment:
Nothing wrong here but I remember you told me we don't need the region stuff with v2 so just curious as to why you put it back in.

Reply:
I was getting an error, It couldn't find a region declared in my settings so i just added it explicitly.